### PR TITLE
StarWarsSample: Xamarin.Forms version

### PR DESCRIPTION
**Pull request by nmilcoff**
@FabriBertani and I have been working on a Xamarin.Forms version of StarWarsSample!

It works on Android and iOS. All ViewModels are implemented, so the app is quite similar 🙂

### CODE REVIEW

File: StarWarsSample/StarWarsSample.Forms/StarWarsSample.Forms.Android/StarWarsSample.Forms.Droid.csproj [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>bin\Debug</OutputPath>
+    <DefineConstants>DEBUG;</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <AndroidHttpClientHandlerType>Xamarin.Android.Net.AndroidClientHandler</AndroidHttpClientHandlerType>
+    <AndroidTlsProvider>btls</AndroidTlsProvider>
+    <AndroidSupportedAbis>x86</AndroidSupportedAbis>
```
Comment (Cheesebaron) [inline StarWarsSample/StarWarsSample.Forms/StarWarsSample.Forms.Android/StarWarsSample.Forms.Droid.csproj]: Please remove this line so people don't have to spend time figuring out why the app doesn't run on the device.

**Comment by Cheesebaron** [comment]
Looks good, ran it on Android, din't have my mac near me.

I've added some rules to CodeFactor so next analysis pass should ignore the LinkerPleaseInclude files it has false positives on.

**Comment by nmilcoff** [comment]
Requested change was applied :)

**Comment by Cheesebaron** [comment]
No just remove the line entirely 😄

**Comment by nmilcoff** [comment]
Haha got it. Done!

**Pull request merged**